deploy/openshift/sources/link_multus_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Maybe you could add a comment on why this is done (in case of reboot, /var/run is wiped.... etc)?

Reply:
Done